scripts/common.sh

```diff
@@ -74,6 +67,20 @@ function common::run_xline() {
         cmd="${cmd} --is-leader"
     fi
 
-    docker exec -e RUST_LOG=info -d node${1} ${cmd}
-    log::info "command is: docker exec -e RUST_LOG=debug -d node${1} ${cmd}"
+    if [ -n "$RUST_LOG" ]; then
+        docker_cmd="docker run -e RUST_LOG=${RUST_LOG} -d -it --rm --name=node${ith} --net=xline_net \
+            --ip=${SERVERS[$ith]} --cap-add=NET_ADMIN --cpu-shares=1024 \
+            -m=512M ${mount_point} ${XLINE_IMAGE} ${cmd}"
+    else
+        docker_cmd="docker run -d -it --rm --name=node${ith} --net=xline_net \
+            --ip=${SERVERS[$ith]} --cap-add=NET_ADMIN --cpu-shares=1024 \
+            -m=512M ${mount_point} ${XLINE_IMAGE} ${cmd}"
+    fi
```

Comment:
If the `RUST_LOG` is the only difference, please don't duplicate the whole command. Would you please find a way to deduplicate?

Reply:
Done

Reply:
Done